README.md

```diff
@@ -1,6 +1,209 @@
+
+The goal of the project is to enable an evaluation of the effects for single and multi-spacecraft architectures when applied to a prospective mission with specified science objectives. In particular, TOSS is designed to be:
+
+- **open-source**: the source code of TOSS is publicly available.
+- **decentralised**: each function and capability within the module can be adopted to generate trajectories for a wider variety of objectives and celestial bodies.
+- **user-expandable**: the user can expand the capabilities of the code using the decentralised structure. 
+
+The optimization is based on an evolutionary approach sampling a large population where each candidate solution results in a deterministics trajectory that is obtained by numerical intergration. The trajectories are then resamples for a desired fixed time-step and then evaluating according to a defined fitness function. Then, unless the algorithm has met a certain stopping criteria, the population is mutated accoridng to the best candidate solution and the optimisation continuous by reiterating the process again.
+
+<p align="center">
+  <a href="https://github.com/rasmusmarak/TOSS">
+    <img src="docs/source/TOSS_Optimization_Scheme.pdf" alt="Scheme"  width="910" height="459">
+  </a>
+  <p align="center">
+    Trajectory Optimisation of a Spacecraft Swarm
+    <br />
+    <a href="https://github.com/rasmusmarak/TOSS/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rasmusmarak/TOSS/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+### Built With
+
+This project is based on:
+
+- [polyhedral-gravity-model](https://github.com/esa/polyhedral-gravity-model) a fast, parallelised version of the polyhedral gravity model implemented in C++ along with a python interface.
+- [DESolver](https://github.com/Microno95/desolver) a python library fro solving Initial Value Problems using numerical integrators. 
+- [PyGMo](https://esa.github.io/pygmo2/index.html) a scientific python library for massively parallel optimization managed by the Advanced Concepts Team at the European Space Agency
+
+For more details than provided by TOSS on these libraries, please refer to their docs.
+
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This is a brief guide how to set up TOSS.
+
+### Installation
+
+### conda
+The recommended way to install TOSS is via [conda](https://docs.conda.io/en/latest/) / [mamba](https://github.com/conda-forge/miniforge#mambaforge) using
+
+```
+conda install toss -c conda-forge
+
+```
```

Comment:
?? I don't think we are on conda

Reply:
if you want to go in that direction I would suggest we open an issue to add pip first as that is way easier but would not write either here until it is actually possible

Reply:
I removed this for now, but have added a new issue for this at #56.